source/geometry/FloatPoint3D.ooc

```diff
@@ -17,7 +17,7 @@ FloatPoint3D: cover {
 	x, y, z: Float
 
 	norm ::= (this x squared + this y squared + this z squared) sqrt()
-	normalized ::= this / this norm
+	normalized ::= (this x equals(0.0f) && this y equals(0.0f) && this z equals(0.0f)) ? (this as FloatPoint3D) : (this / this norm)
```

Comment:
@emilwestergren there is one already for `FloatVector3D` but rock fails if I use it here instead of this hard coded check.